packages/contracts-bedrock/CONTRIBUTING.md

```diff
@@ -61,20 +62,20 @@ Optimism's smart contracts are written in Solidity and we use [foundry](https://
 
 Our [Style Guide](STYLE_GUIDE.md) contains information about the project structure, syntax preferences, naming conventions, and more. Please take a look at it before submitting a PR, and let us know if you spot inconsistencies!
 
-Once you've read the styleguide and are ready to work on your PR, there are a plethora of useful `pnpm` scripts to know about that will help you with development:
-1. `pnpm build` Builds the smart contracts.
-1. `pnpm test` Runs the full `forge` test suite.
-1  `pnpm gas-snapshot` Generates the gas snapshot for the smart contracts.
-1. `pnpm semver-lock` Generates the semver lockfile.
-1. `pnpm storage-snapshot` Generates the storage lockfile.
-1. `pnpm autogen:invariant-docs` Generates the invariant test documentation.
-1. `pnpm clean` Removes all build artifacts for `forge` and `go` compilations.
-1. `pnpm validate-spacers` Validates the positions of the storage slot spacers.
-1. `pnpm validate-deploy-configs` Validates the deployment configurations in `deploy-config`
-1. `pnpm slither` Runs the slither static analysis tool on the smart contracts.
-1. `pnpm lint` Runs the linter on the smart contracts and scripts.
-1. `pnpm pre-pr` Runs most checks, generators, and linters prior to a PR. For most PRs, this is sufficient to pass CI if everything is in order.
-1. `pnpm pre-pr:full` Runs all checks, generators, and linters prior to a PR.
+Once you've read the styleguide and are ready to work on your PR, there are a plethora of useful `just` scripts to know about that will help you with development:
+1. `just build` Builds the smart contracts.
+1. `just test` Runs the full `forge` test suite.
+1  `just gas-snapshot` Generates the gas snapshot for the smart contracts.
+1. `just semver-lock` Generates the semver lockfile.
+1. `just storage-snapshot` Generates the storage lockfile.
+1. `just autogen-invariant-docs` Generates the invariant test documentation.
+1. `just clean` Removes all build artifacts for `forge` and `go` compilations.
+1. `just validate-spacers` Validates the positions of the storage slot spacers.
+1. `just validate-deploy-configs` Validates the deployment configurations in `deploy-config`
+1. `just slither` Runs the slither static analysis tool on the smart contracts.
```

Comment:
looks like we never had a command in the package.json to run slither either, though there are instructions for it in the readme
```suggestion
```